pex/pex_bootstrapper.py

```diff
@@ -118,6 +118,13 @@ def maybe_reexec_pex(pex_info):
 
   current_interpreter = PythonInterpreter.get()
 
+  # NB: Used only for debugging and tests.
```

Comment:
I'm not happy with this, but its the best idea I could come up with.

Reply:
I still think it shouldn't exist so I'd like to keep only. We do already log all this! See:
+ re-exec:
  + https://github.com/pantsbuild/pex/pull/741/files#diff-670bc388096d1ef0c7994c4d37976392R163
+ actual used:
  + https://github.com/pantsbuild/pex/pull/741/files#diff-670bc388096d1ef0c7994c4d37976392R149
  + https://github.com/pantsbuild/pex/pull/741/files#diff-670bc388096d1ef0c7994c4d37976392R157

And I refuse to scrape logs in tests - an even worse solution imo!